packages/core/src/Page/PageVueServerRenderer.ts

```diff
@@ -1,15 +1,25 @@
-const Vue = require('vue');
-const path = require('path');
-const fs = require('fs-extra');
-
-const VueCompiler = require('vue-template-compiler');
-const { renderToString } = require('vue-server-renderer').createRenderer();
+/* eslint-disable import/no-import-module-exports */
+import Vue from 'vue';
+import path from 'path';
+import fs from 'fs-extra';
+import * as VueCompiler from 'vue-template-compiler';
+import { createRenderer } from 'vue-server-renderer';
+import * as logger from '../utils/logger';
+import { PageConfig, PageAssets } from './PageConfig';
+/* eslint-enable import/no-import-module-exports */
 
 let bundle = require('@markbind/core-web/dist/js/vueCommonAppFactory.min');
 
-const logger = require('../utils/logger');
+const { renderToString } = createRenderer();
+
+interface PageEntry {
+  page : any;
```

Comment:
I'm surprised the airbnb base config dosen't enforce this, let's follow `var: type` spacing to be consistent with the other files.

It's presumably a small change so feel free to add this rule into our eslint configs here too if you'd like. Or we can create a follow up.

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/type-annotation-spacing.md

Reply:
I've updated the eslint rules since it's a pretty small change. Thanks for pointing this out!